docs/concepts/configuration/assign-pod-node.md

```diff
@@ -315,20 +316,15 @@ web-server-1287567482-6f7v5    1/1       Running   0          7m        10.192.4
 web-server-1287567482-s330j    1/1       Running   0          7m        10.192.3.2   kube-node-2
 ```
 
-Best practice is to configure these highly available stateful workloads such as redis with AntiAffinity rules for more guaranteed spreading.
```

Comment:
did you mean to remove this content?

Reply:
I did. This section is about 'always co-locate', so it made more sense to me to emphasize the anti-affinity part of this example in the beginning of next section instead.